cmd/run-cbcollect.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/couchbaselabs/cbdynclusterd/daemon"
+	"github.com/spf13/cobra"
+)
+
+var cbcollectCmd = &cobra.Command{
+	Use:   "cbcollect <cluster ID>",
+	Short: "Runs cbcollect-info on a cluster",
+	Long:  `Runs cbcollect-info on a cluster`,
+	Run: func(cmd *cobra.Command, args []string) {
+		checkConfigInitialized()
+
+		if len(args) == 0 {
+			printAndExit("No cluster-id provided")
+		}
+
+		path := "/cluster/" + args[0] + "/cbcollect"
+
+		var respData daemon.CBCollectResultJSON
+
+		err := serverRestCall("GET", path, nil, &respData, false)
+
+		if err != nil {
+			fmt.Printf("Failed to run cbcollect-info: %s\n", err)
+			os.Exit(1)
+		}
+
+		for hostname, bytes := range respData.Collections {
+			writeBytes(fmt.Sprintf("cbcollect-%s.zip", hostname), bytes)
```

Comment:
Might be useful to make this configurable via a cli param

Reply:
The output dir? Yeah I'll add that